File: cs/HelloWorld.cs
using System;
public class HelloWorld
{
    public static void Main(){
        System.Console.WriteLine( "Hello, World!");
        dynamic val = 8 * ( (  7 + 8) % 5 + 6 / 2);
        System.Console.WriteLine(val);
       
        String input = Console.ReadLine( );
        try{
                                
            int num = int.Parse( input);
                                                
            for( int i = 0; i < num;i++){
                                                              
                      Console.WriteLine( "Enter the number of times to print \"Yay!\": ");
                                                                
            }
          }catch( FormatException e){
            Console.WriteLine( "You must enter a whole number.");
            }                                                                          
    }
}

File: java/helloworld.java


public class helloworld{
    public static void main(String args[]){
        System.out.println("hello world");
        permutation("","hello");
    }

    public static void permutation(String prefix,String val){
        if(val.length() == 0){
            System.out.println(prefix);
        }else{
            for(int i = 0; i < val.length(); i++){
                
                permutation(prefix + val.charAt(i), val.substring(0,i) + val.substring(i+1, val.length()));
            }
        }
    }

}
